firebase.go

```diff
@@ -126,8 +130,23 @@ func (fb *Firebase) doRequest(method string, body []byte) ([]byte, error) {
 	}
 
 	resp, err := fb.client.Do(req)
-	if err != nil {
+	switch err := err.(type) {
+	default:
 		return nil, err
+	case nil:
+		// carry on
+	case *_url.Error:
```

Comment:
:shrug: I just left that import the way it was. My guess is so that it doesn't conflict with local vars named `url` that are used often here.